<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addAct</class>
 <widget class="QMainWindow" name="addAct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加活动</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: white;
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}
QPushButton{
	background-color: skyblue;
}
QLabel{
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>320</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="name_lab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>60</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>活动名:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="buddy">
        <cstring></cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_le">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入活动名</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="name_lab_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>60</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>开始时间:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="start_dt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_lab_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>60</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>结束时间:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="end_dt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>300</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="add_bt">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_bt">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
